### PR TITLE
Enable nullability in HtmlShim

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Enable nullability in HtmlShim.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9488)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs [hunk 1]
```diff
@@ -31,9 +29,9 @@ private EventHandlerList Events
     {
         get
         {
-            events ??= new EventHandlerList();
+            _events ??= new EventHandlerList();
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs:32]: nit: expressional body?

File: src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs [hunk 2]
```diff
@@ -98,17 +96,17 @@ protected virtual void Dispose(bool disposing)
         if (disposing)
         {
             DisconnectFromEvents();
-            if (events is not null)
+            if (_events is not null)
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs:99]: _events?.Dispose();
_events = null;

Comment (gpetrou) [reply src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs:99]: In that case we would always set to null even if it is already null, right?

Comment (dreddy-work) [reply src/System.Windows.Forms/src/System/Windows/Forms/HtmlShim.cs:92]: yes. that should be fine.

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**